artacle-swagger.yaml

```diff
@@ -22,6 +22,8 @@ tags:
   - name: Profiles
   - name: Traits
```

Comment:
I'v thought we could rather remove the description here to aling the representation with the other blocks